src/style.css

```diff
@@ -0,0 +1,137 @@
+html {
+  font-family: "Roboto", sans-serif;
+
+  --color-button: #00acdc;
+  --color-main-text: #060b35;
+  --horizontal-padding: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 40px;
```

Comment:
Please add paddings for card  instead adding paddings for each elements. It will be smth like this:
![image](https://user-images.githubusercontent.com/54936044/224659662-16ec6e49-51db-411a-abf8-95d0ada3c160.png)